Valyreon.Elib.Wpf/App.xaml.cs

```diff
@@ -22,8 +23,17 @@ public partial class App : Application
         public static extern void FreeConsole();
 #endif
 
+        private static Mutex _mutex = null;
+
         private void OnStartup(object sender, StartupEventArgs e)
         {
+            _mutex = new Mutex(true, GetType().Namespace.ToString(), out var appInstance);
```

Comment:
I think you also need to release the Mutex on app shutdown.

Reply:
Yes, you're correct. I'll add the release in `OnExit` method.